packages/app/src/__deprecated__/renderer/components/core/tooltip/element-with-tooltip.component.tsx

```diff
@@ -101,6 +105,26 @@ const ElementWithTooltip: FunctionComponent<Props> = ({
     }
   }
 
+  useLayoutEffect(() => {
+    const trigger = () => {
+      if (showIfTextEllipsis) {
+        const text = ref.current as HTMLDivElement
+        const isEllipsisActive = text && text.offsetWidth < text.scrollWidth
+        setIsEllipsis(isEllipsisActive ? isEllipsisActive : false)
```

Comment:
Wouldn't that be enough?

```suggestion
        setIsEllipsis(isEllipsisActive)
```

Reply:
Changed in [30aa052](https://github.com/mudita/mudita-center/pull/1030/commits/30aa0526662289c0a91e42b79d17d013c80a0236)